test/testcafe/spec/OktaSignIn_spec.js

```diff
@@ -0,0 +1,58 @@
+import { ClientFunction, RequestMock, userVariables } from 'testcafe';
+import { checkA11y } from '../framework/a11y';
+import IdentityPageObject from '../framework/page-objects/IdentityPageObject';
+import xhrIdentifyWithPassword from '../../../playground/mocks/data/idp/idx/identify-with-password';
+
+const identifyMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrIdentifyWithPassword);
+
+const hideWidget = ClientFunction(() => {
+  // function `getWidgetInstance` is defined in playground/main.js
+  window.getWidgetInstance().hide();
+});
+
+const showWidget = ClientFunction(() => {
+  // function `getWidgetInstance` is defined in playground/main.js
+  window.getWidgetInstance().show();
+});
+
+const renderHiddenWidget = ClientFunction(({v3}) => {
+  // functions `renderPlaygroundWidget` and `getWidgetInstance` are defined in playground/main.js
+  window.renderPlaygroundWidget();
+  if (v3) {
+    window.getWidgetInstance().hide();
+  } else {
+    window.getWidgetInstance().on('ready', () => {
+      window.getWidgetInstance().hide();
+    });
+  }
```

Comment:
Can you add a comment with this ticket number: https://oktainc.atlassian.net/browse/OKTA-637702 here to update this test to remove the conditional statement. Currently G3 does not trigger the ready event but it should, so this conditional statement should be removed once we add that implementation.